ansible/roles/ocp4-workload-sso-cloud-native-apps/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+become_override: False
+ocp_username: opentlc-mgr
+silent: False
+
+num_users: 5
+workshop_openshift_user_name: userXX
```

Comment:
Variables should be prefixed with the workload name. In this example `ocp4_workload_sso_cloud_native_apps_.....`. So for example `ocp4_workload_sso_cloud_native_apps_user_name`.